application/dash_application/views/single_lane_rna.py

```diff
@@ -533,6 +535,17 @@ def update_pressed(click,
             failure_df.to_dict('records'),
             df.to_dict("records", into=dd),
         ]
+
+    @dash_app.callback(
+        [Output(ids["approve-run-button"], "href"),
+         Output(ids["approve-run-button"], "style")],
+        [Input(ids["run-id-list"], "value")]
+    )
+    def approve_run(value):
+        if len(value) == 1:
+            return ["https://miso.oicr.on.ca/miso/run/alias/" + value[0], {"display": "inline-block"}]
+        else:
+            return ["", {"display": "none"}]
```

Comment:
If you wanted, you could move this functionality into sidebar_utils and use one method everywhere.